python/kserve/Makefile

```diff
@@ -5,7 +5,7 @@ dev_install:
 	pip install -e .[test]
 
 test:
-	pytest -W ignore
+	cd ../ && pytest -W ignore kserve/test
```

Comment:
currently broken if you run `make test` but changing the path in the file will break the other test commands.